core/src/main/java/com/xmartlabs/base/core/Injector.java

```diff
@@ -1,11 +1,13 @@
 package com.xmartlabs.base.core;
 
 import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
 
 import bullet.ObjectGraph;
 
 public class Injector {
```

Comment:
Because I don't know if the name "component" is right here, considering the dagger context.